innopoints/views/activity.py

```diff
@@ -10,6 +26,7 @@
 from innopoints.schemas import ActivitySchema, CompetenceSchema
 
 NO_PAYLOAD = ('', 204)
+log = logging.getLogger(__name__)
```

Comment:
convention is to name it `logger`. "log" is an action verb, which implies a function not an object.

Reply:
We've talked about this) I don't like that, and with `log` the calls like `log.info()` make a lot more sense